### PR TITLE
Superscript UI component

### PR DESCRIPTION
**Pull request by youngkidwarrior**

### CODE REVIEW

**Comment by youngkidwarrior** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

Current dependencies on/for this PR:
* `main`
  * **PR #80** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/80?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #81** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/81?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
      * **PR #83** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/83?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
        * **PR #84** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/84?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
          * **PR #87** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/87?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
            * **PR #92** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈
              * **PR #93** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/93?utm_source=stack-comment-icon" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/92?utm_source=stack-comment).

File: packages/ui/src/components/SuperScript.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  }
+) => {
+  const { fontSize: fontSizeToken, fontWeight, color, children, ...rest } = props
+
+  const fontSize =
+    typeof fontSizeToken === 'string' ? getTokens().radius[fontSizeToken].val : fontSizeToken
+
+  const superFontSize = Math.floor(fontSize * 0.6)
+
+  const superlineHeight = superFontSize * 1.1 // Make sure the superscript lineheght always matches the height of the text
```
Comment (0xBigBoss) [inline packages/ui/src/components/SuperScript.tsx:15]: Ah very cool. Any screenshots of it's usage would be cool.

**Comment by youngkidwarrior** [comment]
The `TM` is a superscript

**Review by 0xBigBoss — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Feb 7, 3:08 AM EST**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/92).

**Pull request merged**